### ISSUE TITLE
3 suspicious URLs

### ISSUE THREAD

**Issue by momo-fr** [elementor/elementor#7673]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Hi, the last analysis of Wordfence up 3 suspicious URLs related to Google, I guess it's the fact that they are not in HTTPs but I'm not sure.

## Steps to reproduce

Scan with Wordfence :
File contains suspected malware URL: wp-content/plugins/elementor-pro/assets/lib/social-share/social-share.js
This file contains a URL that is currently listed on Wordfence's domain blacklist. The URL is: http://plus.google.com/share

File contains suspected malware URL: wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/contact-url.php
This file contains a URL that is currently listed on Wordfence's domain blacklist. The URL is: http://www.google.com/calendar/render

File contains suspected malware URL: wp-content/plugins/elementor-pro/modules/forms/classes/recaptcha-handler.php
This file contains a URL that is currently listed on Wordfence's domain blacklist. The URL is: http://www.google.com/recaptcha/

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.6.42-84.2
	PHP Version: 7.2.16
	PHP Max Input Vars: 1000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Not connected (HTTP Error (503))

== WordPress Environment ==
	Version: 5.1.1
	Site URL: https://www.vacanzanimaux.fr
	Home URL: https://www.vacanzanimaux.fr
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: fr-FR
	Timezone: 1
	Debug Mode: Inactive

== Theme ==
	Name: VacanZanimaux
	Version: 2.0
	Author: FreePixel
	Child Theme: Yes
	Parent Theme Name: OceanWP
	Parent Theme Version: 1.6.4
	Parent Theme Author: Nick

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:66.0) Gecko/20100101 Firefox/66.0

== Active Plugins ==
	Akismet Anti-Spam
		Version: 4.1.1
		Author: Automattic

	BackWPup
		Version: 3.6.8
		Author: Inpsyde GmbH

	Black Studio TinyMCE Widget
		Version: 2.6.8
		Author: Black Studio

	Elementor
		Version: 2.5.10
		Author: Elementor.com

	Elementor Pro
		Version: 2.5.1
		Author: Elementor.com

	Flow-Flow Lite
		Version: 3.1.0
		Author: Looks Awesome

	Granular Controls Elementor
		Version: 1.0.5
		Author: Zulfikar Nore

	Imagify
		Version: 1.8.4.1
		Author: WP Media

	MainWP Child
		Version: 3.5.6
		Author: MainWP

	myStickymenu
		Version: 2.0.6
		Author: m.r.d.a

	Ocean Extra
		Version: 1.5.3
		Author: OceanWP

	Redirection
		Version: 4.1.1
		Author: John Godley

	Safe SVG
		Version: 1.9.3
		Author: Daryll Doyle

	Wordfence Security
		Version: 7.2.4
		Author: Wordfence

	WP Rocket
		Version: 3.2.6
		Author: WP Media

	Yoast SEO
		Version: 10.0.1
		Author: Team Yoast


== Must-Use Plugins ==
	No french punctuation and accents for filename
		Version: 1.0
		Author: BeAPI


== Log ==
	: 
Log: showing 20 of 372019-03-16 17:19:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.7',
  'to' => '2.5.8',
)]
2019-03-19 12:17:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.8',
  'to' => '2.5.9',
)]
2019-03-26 20:37:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.9',
  'to' => '2.5.10',
)]
2019-03-26 20:37:39 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.8',
  'to' => '2.5.1',
)]
2019-03-26 20:37:39 [info] elementor-pro::elementor_pro_updater Started 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_form Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_form Finished 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_woocommerce_menu_cart Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_woocommerce_menu_cart Finished 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Finished 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_portfolio Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_portfolio Finished 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_products Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_products Finished 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_sitemap Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_sitemap Finished 
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Start  
2019-03-26 20:37:39 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Finished 
2019-03-26 20:37:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.8',
  'to' => '2.5.1',
)]
```
</details>

**Comment by shilo-ey** [comment]
Hi @momo-fr 

Thanks for reporting.

In Elementor Pro 2.5.2 this issue only applies to Google Calendar URL, and it will be fixed in the next version.

Thanks again!

**Comment by shilo-ey** [comment]
This issue has been fixed in Elementor Pro v2.5.3

Feel free to update!
